pavelib/quality.py

```diff
@@ -470,6 +516,17 @@ def _pep8_output(count, violations_list, is_html=False):
     ):
         diff_quality_percentage_pass = False
 
+    # run diff-quality for safelint.
+    if not run_diff_quality(
+            violations_type="safelint",
```

Comment:
@nedbat: I forwarded you an email.  It seems that diff-cover now can work with other tools.  I'm going to discuss with @benpatterson and @jzoldak end of week.